daprdocs/content/en/reference/api/actors_api.md

```diff
@@ -237,6 +239,25 @@ To configure the reminder to fire only once, the period should be set to empty s
 }
 ```
 
+When you specify the repitiion number in both `period` and `ttl`, the timer/reminder will be stopped when either condition is met. The following example has a timer with a `period` of 3 seconds (in ISO 8601 duration format) and a `ttl` of 20 seconds. This timer fires immediately after registration, then every 3 seconds after that for the duration of 20 seconds.
```

Comment:
```suggestion
When you specify the repetition number in both `period` and `ttl`, the timer/reminder is stopped when either condition is met. The following example has a timer with a `period` of 3 seconds (in ISO 8601 duration format) and a `ttl` of 20 seconds. This timer fires immediately after registration, then every 3 seconds after that for the duration of 20 seconds, after which it never fires again since the `ttl` was met
```

Reply:
@JoshVanL - Can `ttl `and `dueTime` be used together? Or are these independent?

Reply:
If only ttl and dueTime are set, the reminder will be accepted however only the dueTime will take effect- i.e. the reminder will trigger at dueTime and ttl is ignored.

Reply:
Thanks. I suggest that we add this into the doc, so that this is clear. 
@hhunter-ms - Could you add these sentences?